database/migrations/sqls/20210513114937-addCaptureTakenColumn-up.sql

```diff
@@ -1,2 +1,2 @@
 /* Replace with your SQL commands */
-ALTER TABLE raw_capture ADD capture_taken_at timestamptz NOT NULL;
\ No newline at end of file
+ALTER TABLE raw_capture ADD capture_taken_at timestamptz;
```

Comment:
This file could have had all three statements together. But this is fine as well.